shipshape.yml

```diff
@@ -219,6 +219,9 @@ checks:
   drupal-admin-user:
     - name: '[DATABASE] Active user roles admin check'
       severity: high
+  drupal-user-forbidden:
+    - name: '[DATABASE] Active User 1 check'
+      severity: high
```

Comment:
After chatting with Steve, we might want to leave this check at `normal` severity so that it does not block deployments initially.

```suggestion
```